### PR TITLE
15 stanage hpc missing from readme

### PR DESCRIPTION
**Pull request by gucci-j**
This PR fixes typos and adds further descriptions regarding `setup_ssl_ca_server_client.sh`

### CODE REVIEW

**Review by nikishe — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ This will setup the local configuration (including a password for you) and store
 
 After the server has fully started, terminate it by pressing Ctrl+C .
 
-Now you should generate your SSL certificates (these secure the communications between your local device and the endpoint worker node running the vscode server.)
```
Comment (jkwmoore) [inline README.md:138]: This change isn't right if recall correctly. This script is provided on system as part of the module you would have loaded at this stage of the instructions.

So there is no need to download the script and run it.

If this is missing from the module it should be added.

**Comment by jkwmoore** [comment]
@Carldkennedy do you think we can fold these changes into the exist PR and get that reviewed (I am a bit short on time) via other support team members?

File: README.md (hunk 1 above)
Comment (gucci-j) [reply README.md]: Thanks! I have just reverted the description as I have confirmed that the script is automatically loaded when the vscode-server module is loaded.

**Pull request merged**